freeipa/src/main/resources/application.yml

```diff
@@ -64,6 +64,9 @@ freeipa:
   environment:
     url: http://localhost:8088
     contextPath: /environmentservice
+  sdx:
```

Comment:
This is a new dependency of FreeIPA, thus we need to introduce it in CBD and helm-charts as well.